napi-derive/src/lib.rs

```diff
@@ -30,6 +30,7 @@ struct JsFunction {
   args: Vec<FnArg>,
   name: mem::MaybeUninit<Ident>,
```

Comment:
Why isn't `Option` used here for `name`, `signature`, and `signature_raw`?

Reply:
Oh, it's because I don't like `unwrap` before, but I think it's better to change to `Option` now